go.mod

```diff
@@ -2,6 +2,7 @@ module github.com/elastic/beats/v7
 
 go 1.21
 
+// replace github.com/elastic/elastic-agent-autodiscover => /Users/michaliskatsoulis/go/src/github.com/elastic/elastic-agent-autodiscover
```

Comment:
Why is this comment present? Seems like it should be removed.

Reply:
Thanks, it is a mistake, I will remove it